### PR TITLE
Add basic poll related classes and GUI

### PR DESCRIPTION
**Pull request by MuhdNurKamal**
Add classes for basic voting system
Add poll GUI
Add read/write support for polls

### CODE REVIEW

**Comment by MuhdNurKamal** [comment]
PR for task #118

File: src/main/java/seedu/club/model/poll/Answer.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+import static java.util.Objects.requireNonNull;
+import static seedu.club.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents an answer to a question of a poll
+ */
+public class Answer {
+
+    public static final String MESSAGE_ANSWER_CONSTRAINTS = "Answer not be empty";
```
Comment (yash-chowdhary) [inline src/main/java/seedu/club/model/poll/Answer.java]: "Answer cannot be empty"*

File: src/main/java/seedu/club/model/poll/Answer.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    public static final String MESSAGE_ANSWER_NUMBER_ANSWERED_CONSTRAINTS =
+            "Number answered for an answer should be non-negative";
```
Comment (yash-chowdhary) [inline src/main/java/seedu/club/model/poll/Answer.java]: "Invalid option selected" or "Invalid option number selected" ?

**Review by yash-chowdhary — COMMENTED** [review]
Looks good! Couple of minor changes.

**Comment by yash-chowdhary** [comment]
Just to confirm, the polls are shown in the middle panel of the GUI, right?

File: src/main/java/seedu/club/model/poll/Answer.java (hunk 1 above)
Comment (MuhdNurKamal) [reply src/main/java/seedu/club/model/poll/Answer.java]: Ops. Thanks

File: src/main/java/seedu/club/model/poll/Answer.java (hunk 2 above)
Comment (MuhdNurKamal) [reply src/main/java/seedu/club/model/poll/Answer.java]: This message is not for picking of wrong answer. This string is actually for construction of Answer with number of members that answered/picked this answer. For normal construction it's left blank(indicate 0) but from reading from storage i need to test that the number of members that picked this answer is non-zero.

Comment (yash-chowdhary) [reply src/main/java/seedu/club/model/poll/Answer.java]: Oh I see. Then how about displaying "An answer can only be chosen by >=0 members" or "An  answer cannot be picked negative number of times" ?

**Comment by MuhdNurKamal** [comment]
Yup, just to the left of the task pane
